views/components/Sidebar/SideNavigation.css

```diff
@@ -0,0 +1,5 @@
+.SideNavigation_nav {
+  height: 664px;
+  padding-left: 6px;
+  padding-right: 12px;
```

Comment:
I don't see where the designs have a larger right padding than left.

Reply:
[377af01](https://github.com/AMP-SCZ/dpdash/pull/458/commits/377af0123d164798059bf0dc152683f783a6b3f8)